### PR TITLE
Dh calendar page

### PR DESCRIPTION
**Pull request by Dimo-Hristov**
1. Make title text dynamic.
2. Fix calendar page styles.

### CODE REVIEW

File: src/app/calendar/calendar-list/calendar-list.component.ts [hunk 1]
```diff
@@ -35,5 +41,9 @@ export class CalendarListComponent implements OnInit, OnChanges {
         return this.selectedDate === currentDate;
       });
     });
+
+    // Transform the selected date using the BulgarianDateFormatPipe
+    this.initialMonth = this.bulgarianDatePipe.transform(this.selectedDate);
+    this.cdr.detectChanges();
```
Comment (Getsov) [inline src/app/calendar/calendar-list/calendar-list.component.ts]: this is very dangerous, why do we need it and don't we have another ways for achieving the wanted result?

File: src/app/calendar/calendar.page.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { EventsService } from 'src/shared/services/events.service';
   templateUrl: './calendar.page.html',
   styleUrls: ['./calendar.page.scss'],
 })
-export class CalendarPage {
+export class CalendarPage implements OnInit {
```
Comment (Getsov) [inline src/app/calendar/calendar.page.ts]: why implemented but empty?

File: src/app/calendar/calendar.page.ts [hunk 3]
```diff
@@ -26,14 +26,18 @@ export class CalendarPage {
     private datePipe: DatePipe
   ) {}
 
+  ngOnInit(): void {}
```
Comment (Getsov) [inline src/app/calendar/calendar.page.ts]: Here is the empty ngOnInit ?

File: src/app/calendar/dates/dates.component.scss [hunk 4]
```diff
@@ -8,24 +8,29 @@ ion-datetime {
...
     color: black;
   }
+  --ion-color-step-300: rgba(166, 162, 160, 1);
+  --ion-color-light-rgb: none;
 }
 
 ion-datetime::part(today) {
   color: #fff !important;
-  font-weight: 600 !important;
+  font-weight: 00 !important;
```
Comment (Getsov) [inline src/app/calendar/dates/dates.component.scss]: 00 ?

File: src/app/calendar/calendar-list/calendar-list.component.ts (hunk 1 above)
Comment (Dimo-Hristov) [reply src/app/calendar/calendar-list/calendar-list.component.ts]: i forgot to remove it, it was for testing. Removed.

File: src/app/calendar/calendar.page.ts (hunk 2 above)
Comment (Dimo-Hristov) [reply src/app/calendar/calendar.page.ts]: removed

File: src/app/calendar/calendar.page.ts (hunk 3 above)
Comment (Dimo-Hristov) [reply src/app/calendar/calendar.page.ts]: removed

**Review by Getsov — APPROVED** [review]

**Pull request merged**